yt/funcs.py

```diff
@@ -1332,3 +1332,15 @@ def levenshtein_distance(seq1, seq2, max_dist=None):
         if matrix[x].min() > max_dist:
             return max_dist + 1
     return matrix[size_x - 1, size_y - 1]
+
+
+def validate_moment(moment, weight_field):
```

Comment:
not sure `yt.funcs` is a good home for this function. Do you expect it to be adopted downstream ? if not, I'd suggest putting it in a more private space

Reply:
Do we have a standard for what should go in here? I'm honestly not certain of where else it should go. I'd obviously not want to create a single file for it.

Reply:
never mind, I remembered that we had a couple other `validate_*` methods but turns out they also live in this module. I've never been satisfied with how this module hosts both internal and user-facing functions but I guess it's a bit late to change it and it's certainly out of scope here